packages/a/antd-mobile.json

```
@@ -24,7 +24,7 @@
     "target": "antd-mobile",
     "fileMap": [
       {
-        "basePath": "dist",
+        "basePath": "bundle",
         "files": [
           "**/!(*.less)"
```

Looking at the output from the CI, I think this can be reduced to something far safer now:

```suggestion
          "*.@(js|css)"
```